pkg/operators/formatters/formatters.go

```diff
@@ -325,6 +341,44 @@ var replacers = []replacer{
 		priority: 0,
 	},
 	{
+		name:      "bytes",
+		selectors: []string{"type:" + ebpftypes.BytesTypeName},
+		replace: func(logger logger.Logger, ds datasource.DataSource, in datasource.FieldAccessor) (func(data datasource.Data) error, error) {
+			outName, err := annotations.GetTargetNameFromAnnotation(logger, "formatters.bytes", in, bytesTargetAnnotation)
+			if err != nil {
+				return nil, err
+			}
+
+			opts := []datasource.FieldOption{
+				datasource.WithAnnotations(map[string]string{
+					metadatav1.TemplateAnnotation: "bytes",
```

Comment:
The `bytes` template doesn't exist. It should be added to pkg/metadata/v1/annotations.go.